### DISCUSSION TITLE
Bug: Toast always closes all messages

### DISCUSSION THREAD

**Discussion by tweiler0309** [primefaces/community#1730] [PrimeReact] [answered]
Closing one of several messages closes all messages. Independent of their sticky-value. This is a breaking behaviour opposit to PrimeReact 9.6.5. This behaviour can be reproduced in both official demo-applications of PrimeReact (10.6.3 vs. 9.6.5)
Closing a single message and leave all other messages open is important.
How to deal with it in 10.6.5?
Thx guys!

**Comment by melloware** [comment]
Open a bug ticket on the https://github.com/primefaces/primereact/issues

**Reply by melloware** [reply, marked as answer]
Fixed for next release: https://github.com/primefaces/primereact/pull/6457
